mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6132 applyArmorToDamage (Lnet/minecraft/class_1282;F)F
 		ARG 1 source
 		ARG 2 amount
+	METHOD method_23883 dropXp ()V
+	METHOD method_23733 onKilledBy (Lnet/minecraft/class_1309;)V
+		COMMENT Performs secondary effects after this mob has been killed.
+		COMMENT 
+		COMMENT <p> The default behaviour spawns a wither rose if {@code adversary} is a {@code WitherEntity}.
+		COMMENT 
+		COMMENT @param adversary the main adversary responsible for this entity's death
```

Comment:
Cloak bug? Should be comment on arg 1

Reply:
Yes, you dont need the `@param adversary` bit either

Reply:
Sadly Enigma breaks on LivingEntity, so I can't use it to reformat the file.